python/src/diffusionkit/mlx/mmdit.py

```diff
@@ -209,13 +214,15 @@ def __call__(
         else:
             positional_encodings = None
 
+        timestep_embedding = self.guidance_in(self.t_embedder(timestep))
```

Comment:
Override original timestep only for dev model
```suggestion
	if config.guidance_embed:
            timestep = self.guidance_in(self.t_embedder(timestep))
```